### ISSUE TITLE
Help command in column mode

### ISSUE THREAD

**Issue by polvoazul** [bram85/topydo#166]
It would be nice to have a help command in column mode showing the key shortcuts.
Something like when you press `h` in `less` would be great. I suggest using `:help` as in vim

**Comment by protist** [comment]
Yes please. I frequently have to open up a browser window instead.
